### DISCUSSION TITLE
perguntas

### DISCUSSION THREAD

**Discussion by ranisharma09** [ranisharma09/examplo#3] [Q&A] [answered]
Tenho uma dúvida sobre esta parte do código 'console.log("Hello Word")'

**Comment by suhaniroy121** [comment]
Este é um metodo de print em javascript

**Comment by ranisharma09** [comment]
Yes

**Comment by suhaniroy121** [comment]
Eu vi que tem muitas tecnologias na area de desenvolvimento e gostaria de saber se tem alguma api ou alguém tem uma grade do que eu posso estudar?

**Comment by ranisharma09** [comment, marked as answer]
Existe sim, este repositorio: tem grade de ensino. https://github.com/kamranahmedse/developer-roadmap
